packages/atomic/cypress/e2e/search-box/instant-results/search-box-instant-results-selectors.ts

```diff
@@ -3,6 +3,10 @@ import {SearchBoxSelectors} from '../search-box-selectors';
 export const InstantResultsSelectors = {
   results: () =>
     SearchBoxSelectors.shadow().find('[part~="instant-results-item"]'),
+  activeResult: () =>
+    SearchBoxSelectors.shadow().find(
+      '[part~="active-suggestion"][part~="instant-results-item"]'
+    ),
```

Comment:
Optimize the selection, instead of getting'em'all then checking each, just select/check the active suggestion